desktop-widgets/locationinformation.h

```diff
@@ -5,7 +5,7 @@
 #include "core/units.h"
 #include "core/divesite.h"
 #include "core/taxonomy.h"
-#include "ui_locationinformation.h"
+#include "ui_locationInformation.h"
```

Comment:
this appears to be what breaks all of the compilations on filesystems that actually are case sensitive...